ts/components/icon/SessionIconButton.tsx

```diff
@@ -19,7 +19,7 @@ interface SProps extends SessionIconProps {
   tabIndex?: number;
 }
 
-const StyledSessionIconButton = styled.div<{ color?: string; isSelected?: boolean }>`
+const StyledSessionIconButton = styled.button<{ color?: string; isSelected?: boolean }>`
```

Comment:
Are we 100% sure that changing to button doesn't change any existing styles or layout anywhere? For example I can't remember if a button is a block or inline element but a div is a block element.

Reply:
I am pretty sure it's ok, I had a look already. If really, it shouldn't break anything, just have a style issue until we fix it. And QA might find some

Reply:
Cool. Thanks for checking